### PR TITLE
Update triplea_maps.yaml

### PR DESCRIPTION
**Pull request by RogerCooper**
Adds 2 scenarios using a new map, based on the game Axis & Allies & Zombies

### CODE REVIEW

**Comment by beelee1** [comment]
your version number should be "1". I'm not sure if that's causing the error. Double check your indentation if that doesn't fix it. 

Click on the "Update triplea_maps.yaml" Right above the "Some checks were not successful" and on the upper right there are three dots  will allow you to edit.

**Comment by RogerCooper** [comment]
I don't see 3 dots to edit my branch. When I try to go back in directly, I don't see the branch I created.

image

**Comment by beelee1** [comment]
yea so update triplea_maps. yaml is right above my first post now.

Screenshot from 2021-04-18 21-36-55

Ater you click that it'll take you to

Screenshot from 2021-04-18 21-37-45

The three dots are below "Review Changes" on the right.

File: triplea_maps.yaml [hunk 1]
```diff
@@ -2273,3 +2273,12 @@
     Edition.</p>
     <br><p>Key features include combining UK Pacific and ANZAC into one nation, revised
     order of play for nations, victory objectives and 8 rounds of play.</p>
+ - mapName: AAZ
+   mapCategory: EXPERIMENTAL
+   url: https://github.com/triplea-maps/AAZ/archive/refs/heads/master.zip
+   version: 0
+   img: https://github.com/triplea-maps/AAZ/archive/refs/heads/master/previes.zip
```
Comment (DanVanAtta) [inline triplea_maps.yaml:2280]: The img link should be to the preview image. You can find the link by navigating to the file and then right clicking the donwload button and 'copy link'.
That would be: https://github.com/triplea-maps/AAZ/blob/master/Preview.png

Though, the preview image should also have a lower case 'p' on it.

**Comment by RogerCooper** [comment]
I pushed changes to the game file to change the game names and version number.

**Comment by RogerCooper** [comment]
I am still not seeing how to update the triplea_maps.yaml file. Clicking the url above beelee's post just gets me into a conversation about the yaml file, not the file itself.

**Comment by beelee1** [comment]
hmm...well you can try clicking on your first post and edit that one. 

https://github.com/triplea-game/triplea/pull/9144/commits/c5829ecf6eca0cc570a221d5d561301e34dee22e

Until this is merged it won't show up at 
https://github.com/triplea-game/triplea/blob/master/triplea_maps.yaml

if that's what you mean

**Comment by DanVanAtta** [comment]
FWIW, in 2.6 much of this becomes easier, admittedly this is and has been very convoluted.

Thank you for considering the rename changes and coming up with something. To fit convention, in 2.6 the map repository name should match the map name, for 2.5 and below that is *required*. There are also the game names and XML files that would benefit from a similar naming update. AFAIK this is the list of items that need to be tackled:
- map repository renamed to: zombies_world_war_2
- mapName XML attribute (in each game XML file) renamed to:  zombies_world_war_2
- game XML files renamed
- gameName XML attribute (in each game XML file) renamed
- "Preview.png" in repository renamed to "preview.png" & triplea_maps.yaml entry updated to match

A point of caution: the 'dash' (`-`) might cause problems in the map name, I'm not 100% sure but it is something to watch out for. IIRC anything after a dash is treated as a skin name and everything before is the base map name. The dash in the map name may need to be converted to an underscore.

**Comment by RogerCooper** [comment]
> FWIW, in 2.6 much of this becomes easier, admittedly this is and has been very convoluted.
> 
> Thank you for considering the rename changes and coming up with something. To fit convention, in 2.6 the map repository name should match the map name, for 2.5 and below that is _required_. There are also the game names and XML files that would benefit from a similar naming update. AFAIK this is the list of items that need to be tackled:
> 
> * map repository renamed to: zombies_world_war_2
> * mapName XML attribute (in each game XML file) renamed to:  zombies_world_war_2
> * game XML files renamed
> * gameName XML attribute (in each game XML file) renamed
> * "Preview.png" in repository renamed to "preview.png" & triplea_maps.yaml entry updated to match
> 
> A point of caution: the 'dash' (`-`) might cause problems in the map name, I'm not 100% sure but it is something to watch out for. IIRC anything after a dash is treated as a skin name and everything before is the base map name. The dash in the map name may need to be converted to an underscore.

There are plenty of maps already in the repository which have dashes in their name.

**Comment by DanVanAtta** [comment]
@RogerCooper please let me know if you could use any help or have questions about the list of TODO items, otherwise please comment when they are ready.

**Comment by RogerCooper** [comment]
Is "gameName" a new property replacing "info name"?

**Comment by DanVanAtta** [comment]
Going by memory, I believe it's
```
<info gameName="..." />
```
That property (or more precisely, that attribute of the 'info' tag) is not used in 2.6 and going forward.

**Comment by RogerCooper** [comment]
I don't what you mean by "gameName XML attribute (in each game XML file) renamed" and "That property (or more precisely, that attribute of the 'info' tag) is not used in 2.6 and going forward."

What should I put into my xml file "info name=" or "info gameName="? Or is it both for compatibility with 2.5 & 2.6?

**Comment by DanVanAtta** [comment]
Example:
```
  <info name="Global War" version="1.3"/>
```
The 'version' part of that has never been used beyond UI display. The 'name' part works the same as it has. 2.6 will no longer use the above tag, though we will have users on both 2.5 and 2.6 in parallel, so both versions need to work until we get everyone fully off of 2.5

**Comment by RoiEXLab** [comment]
Closing in favour of #9186

**Pull request closed without merging**